palace/utils/configfile.cpp

```diff
@@ -1702,6 +1706,9 @@ void LinearSolverData::SetUp(json &solver)
   linear->erase("DivFreeTol");
   linear->erase("DivFreeMaxIts");
   linear->erase("GSOrthogonalization");
+
+  linear->erase("EstimatorTol");
```

Comment:
Given where these are parsed, move up to between "DivFreeMaxIts" and "GSOrthogonalization" (style nit).